src/main/resources/data/retail/transactions-currentaccount.json

```diff
@@ -2,7 +2,7 @@
   {
     "category": "Shopping",
```

Comment:
Category should be `Transfer`: https://stash.backbase.com/projects/TRAN/repos/transaction-enrichment-provider/browse/mock-provider/src/main/java/com/backbase/transaction/enrichment/provider/mock/util/MockDataProvider.java#58

Reply:
Thanks for this info. Changed categories.